YOLO.xcodeproj/project.pbxproj

```diff
@@ -13,11 +13,6 @@
 		636EFCAF21E62DD300DE43BC /* VideoCapture.swift in Sources */ = {isa = PBXBuildFile; fileRef = 636EFCA221E62DD300DE43BC /* VideoCapture.swift */; };
 		636EFCB321E62DD300DE43BC /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 636EFCA721E62DD300DE43BC /* AppDelegate.swift */; };
 		636EFCB921E62E3900DE43BC /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 636EFCB821E62E3900DE43BC /* Assets.xcassets */; };
-		6381D2182B7817C200ABA4E8 /* yolov8l.mlpackage in Sources */ = {isa = PBXBuildFile; fileRef = 6381D2132B7817C200ABA4E8 /* yolov8l.mlpackage */; };
-		6381D2192B7817C200ABA4E8 /* yolov8x.mlpackage in Sources */ = {isa = PBXBuildFile; fileRef = 6381D2142B7817C200ABA4E8 /* yolov8x.mlpackage */; };
-		6381D21A2B7817C200ABA4E8 /* yolov8s.mlpackage in Sources */ = {isa = PBXBuildFile; fileRef = 6381D2152B7817C200ABA4E8 /* yolov8s.mlpackage */; };
-		6381D21B2B7817C200ABA4E8 /* yolov8m.mlpackage in Sources */ = {isa = PBXBuildFile; fileRef = 6381D2162B7817C200ABA4E8 /* yolov8m.mlpackage */; };
-		6381D21C2B7817C200ABA4E8 /* yolov8n.mlpackage in Sources */ = {isa = PBXBuildFile; fileRef = 6381D2172B7817C200ABA4E8 /* yolov8n.mlpackage */; };
```

Comment:
In my opinion, these are just references to mlpackage files, and I thought it would be better if they were not there.
If you need them, please let me know and I will restore them and recommit them.
Github does not allow files larger than 100mb in total, so only the mlpackage reference records remain. From experience, I thought that these could cause conflicts when users try this project locally, so I removed them.
Alternatively, we may want to include *.mlpackage in our .gitignore.

Reply:
<img width="248" alt="スクリーンショット 2024-07-26 8 28 16" src="https://github.com/user-attachments/assets/7b644086-127e-413b-aa6c-34e4056a92eb">

Reply:
@john-rocky Let's keep these references so we can easily point out where to place the models during the build phase.

Also, could you add a brief paragraph in the README to clarify that these are just references?

https://github.com/ultralytics/yolo-ios-app/blob/494eedac0528178422cb0e9ab9992c083969e679/README.md?plain=1#L63-L77